ci/hosts

```diff
@@ -1,14 +1,14 @@
 [instance]
 # List of hosts which we want to gather information from
-localhost ansible_connection=local kube_config=/root/.kube/config
+localhost ansible_connection=local metadata_sa=backpack-view
 
 [kubernetes]
 # List of hosts to kick off backpack on all associated nodes
 # Hosts should be defined with a variable definition for 
 # kube_config such as:
 # localhost ansible_connection=local kube_config=/home/user/.kube/config
-localhost ansible_connection=local kube_config=/root/.kube/config
+localhost ansible_connection=local metadata_sa=backpack-view
```

Comment:
if metadata_sa is in the group_vars for kubernetes, we no longer need it in the inventory file.